blog/2023-03-01-effective-focus-indicators/index.md

```diff
@@ -0,0 +1,114 @@
+---
+slug: 2023/01/effective-focus-indicators
+title: Effective and flexible focus indicators
+authors: joe
+tags: [design, libraries, ui]
+---
+
+How can we design effective and flexible focus indicators for a component library?
```

Comment:
We're probably gonna need a longer intro here, so it also displays correct information in the blog post list/homepage. Maybe merge it with the section "What are focus indicators?"